lib/experimental/pattern.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Temporary compatibility shims for pattern APIs present in Gutenberg.
+ *
+ * @package gutenberg
+ */
+
+register_block_pattern(
+	'gutenberg/get-in-touch',
+	array(
+		'title'      => esc_html__( 'Get In Touch', 'default' ),
+		'categories' => array( 'call-to-action' ),
+		'content'    => implode(
+			'',
+			array(
+				'<!-- wp:paragraph {"fontSize":"huge"} -->',
+				'<p class="has-huge-font-size"></$wp:template-content name="getInTouch"></p>',
```

Comment:
Is the `/` in the right place for `$wp:template-content`?

Reply:
I was experimenting with the idea of the tag closer that get converted to the HTML comment in the browser. In effect, if you never replace `</$wp:template-content name="getInTouch">` with the content, the block will still get rendered correctly for the site editors because it will simply skip this syntax for the content placeholder.